app/controllers/application_controller.rb

```diff
@@ -6,7 +6,7 @@ class ApplicationController < ActionController::Base
 
   def mobile_device?
     if session[:mobile_override]
-      session[:mobile_override] == "1"
+      session[:mobile_override] == '1'
```

Comment:
I think we should only declare overrides in that file, not have the entire config. Otherwise it's kind of info overload and we have to worry about keeping that file up to date with upstream.